### ISSUE TITLE
SyntaxError: Unexpected token import

### ISSUE THREAD

**Issue by sean-lin-tw** [filecoin-project/filecoin-client-tutorial#4]
Hi, I am running Node v8.10.0 on Ubuntu18.04. 

When I ran the project with `node server.js`, it showed a syntax error at the `import` statement. 
![image](https://user-images.githubusercontent.com/6895733/86684461-6d2ff000-bfb7-11ea-8aa6-18d23c2d627f.png)
How do you get around this?

Thanks!

**Comment by jimmylee** [comment]
@jacky6016 if you could try upgrading your version of node that might help.

**Comment by sean-lin-tw** [comment]
I upgraded Node to 14.5.0 and change `server.js` to `server.mjs`. It can run now.

**Comment by jeevanpillay0000** [comment]
Doesn't work for me either.

Node - 14.7.0 (LTS)
Npm - 6.14.7

Works if you change to 
```
const express = require('express');
const fs = require("fs");
const { ffs, createPow } = require("@textile/powergate-client")
```
 Also, you need to change the placing of some of the variable instantiations.

**Comment by jimmylee** [comment]
@lpfloyd I don't think this example has been kept up to date, if you could send a PR that would be great
